### PR TITLE
feat(upload): added the delete uploads page

### PR DESCRIPTION
**Pull request by Shruti3004**
## Description

Added the delete Upload file page.

### Changes

- Created the getFolders page from where it is fetching all the folders.
- Created the organizeUploads which contains `deleteUploadFile` and `getUploadsById`.
- Added the property for options in InputContainer.

## How to test

Visit to `/organize/uploads/delete`

## Screenshots
Screenshot from 2021-06-29 08-29-00
Screenshot from 2021-06-29 08-41-18
Screenshot from 2021-06-29 08-42-13

### CODE REVIEW

File: public/thirdPartyLicenses.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
             padding: 0.7em;
             background: white;
             border-top: 1px solid silver;
-            xborder-right: 1px solid silver;
+            border-right: 1px solid silver;
```
Comment (GMishx) [inline public/thirdPartyLicenses.html:45]: Please do not modify this file!

**Review by GMishx — CHANGES_REQUESTED** [review]

**Comment by GMishx** [comment]
Can this list be sorted alphabetically while loading? And once the delete request is successful, please reload the list to prevent duplicate request.

image

File: public/thirdPartyLicenses.html (hunk 1 above)
Comment (Shruti3004) [reply public/thirdPartyLicenses.html:45]: Oh sorry, Actually it is showing error that's why I have fixed it.

Comment (Shruti3004) [reply public/thirdPartyLicenses.html]: @GMishx I have reverted the changes.

**Comment by Shruti3004** [comment]
> Can this list be sorted alphabetically while loading? And once the delete request is successful, please reload the list to prevent duplicate request.
> 
> image

@GMishx I have updated it now duplicate deletion won't happen please check would it work

**Comment by GMishx** [comment]
Something is going wrong here. The page is continuously sending requests. Please have a look.

image

**Comment by Shruti3004** [comment]
@GMishx updated!

**Review by GMishx — APPROVED** [review]
Changes looks good.

**Comment by GMishx** [comment]
Please change the message "Successfully deleted the selected files" to "Successfully scheduled selected uploads for deletion" as it is done by delagent by scheduler and not immediately done. And if user reloads the page again, they can still see the upload which can be confusing.

**Comment by Shruti3004** [comment]
@GMishx Updated as you suggested.

**Pull request merged**